lapis/src/main/kotlin/org/genspectrum/lapis/request/Field.kt

```diff
@@ -9,6 +9,10 @@ data class Field(
 
 fun interface FieldConverter<T> {
     fun convert(source: String): T
+
+    fun validatePhyloTreeFields(source: String) {
+        // Default implementation does nothing, can be overridden if needed
+    }
```

Comment:
Do we really need this method in the interface here? To me it looks like we don't